kivy_ios/recipes/libpng/__init__.py

```diff
@@ -5,24 +5,26 @@
 
 
 class PngRecipe(Recipe):
-    version = '1.6.26'
-    url = 'http://downloads.sourceforge.net/sourceforge/libpng/libpng-{version}.tar.gz'
-    depends = ["python"]
-    library = '.libs/libpng16.a'
+    version = '1.6.40'
+    url = 'https://altushost-swe.dl.sourceforge.net/project/libpng/libpng16/1.6.40/libpng-1.6.40.tar.gz'
```

Comment:
Don't we want to make the `{version}` part of the url?

Reply:
Moved to WIP, as I was supposed to change this URL to a non-mirror one, but I completely forgot. 🤦
Thank you for the heads up! (will handle it in a separate PR before this one gets merged)

(Our download process seems to not correctly handle a change that sourceforge recently performed)